pp/bare_bones/bitset.h

```diff
@@ -66,6 +67,10 @@ class Bitset {
 
   [[nodiscard]] PROMPP_ALWAYS_INLINE size_t capacity() const noexcept { return static_cast<size_t>(data_.size()) * 64; }
 
+  [[nodiscard]] PROMPP_ALWAYS_INLINE size_t empty() const noexcept {
+    return std::ranges::none_of(data_, [](uint64_t v) { return v != 0; });
+  }
```

Comment:
```suggestion
  [[nodiscard]] PROMPP_ALWAYS_INLINE bool empty() const noexcept {
    return std::ranges::all_of(data_, [](uint64_t v) { return v == 0; });
  }
```
all_of is better for readability